Jf delete

jaclynmariefrench opened the pull request:
#### Changes Made

1. Added delete functionally for current logged in users. 
2. Fixed favorites event.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   >make sure the delete function appears for logged in user.
   > make sure favorite button is working. 
  
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.

jaclynmariefrench committed:
delete function added and favorite event fixed

jaclynmariefrench committed:
cleaned up formatting

enbeec commented on src/styles/feed.css:
```diff
@@ -4,3 +4,8 @@
   justify-content: center;
 }
+
+.deleteIcon {
```
did we end up getting rid of this class?

enbeec reviewed (approved):
Just one small thing that we might be able to delete in feed.js

enbeec committed:
Merge branch 'main' into jf-delete

jaclynmariefrench committed:
deleted class in feed.css

jaclynmariefrench committed:
Trying to find local files

Merge branch 'jf-delete' of github.com:nss-day-cohort-48/giffygram-altavista-giffygram into jf-delete
